main.js

```diff
@@ -29,6 +29,13 @@ const importObjectToCollection = async (collection, object, importStats, uniqueK
     sleepPromised(100);
 };
 
+const includeTransformFunction(fnText){
```

Comment:
Did you commit right version?
This probably doesn't work and you do not use includeTransformFunction anywhere.

Reply:
Yes, this is from old version, I don't know how it got here. I'll try removing it.